packages/plugin-testing-integration/src/localization.js

```diff
@@ -43,4 +38,29 @@ function generateDictionary(languages, locale = 'en') {
   return dictionaries;
 }
 
+/**
+ * Apply function through full object or array values
+ *
+ * @param {object|array} obj object to be manipulated
+ * @param {function} fn function to run on values
+ * @returns {object|array}
+ */
+function _deepMapValues(obj, fn) {
+  if (Array.isArray(obj)) {
+    return obj.map(val => _deepMapValues(val, fn));
+  } else if (typeof obj === 'object') {
+    return Object.keys(obj).reduce((acc, current) => {
+      const key = current;
+      const val = obj[current];
+      acc[key] =
```

Comment:
Can't we just do `acc[key] = _deepMapValues(val, fn)`? The object condition is part of the `_deepMapValues`, we can extend the condition on line 51 to return false if obj is `null` and we will get the same results in the end. This way it seems clearer to me, what do you think?

Reply:
Good suggestion, thanks. I had it originally on one line, so I rewrite it for better readability and this simplification didn't occur to me ;)